### PR TITLE
Color Schemes: Add Sakura color scheme

### PR DESCRIPTION
**Pull request by sixhours**
#### Changes proposed in this Pull Request

* Adds a new color scheme, Sakura, to Calypso.

screen shot 2019-02-21 at 4 00 39 pm

#### Testing instructions

* TBD

### CODE REVIEW

**Comment by drw158** [comment]
I'd like to see celadon used instead of the black for the primary color. Overall, the black makes it feel heavy, and "cherry blossom" to me makes me think light and airy.

The black was only added for text accessibility, but we the celadon color can also pass AA, so we have some flexibility.

screen shot 2019-02-20 at 10 13 05 am

I'm not as tied to this idea, but I think it would be interesting to see the light blue used as the selected state for the sidebar item, as shown in the diagram above.

This applies for all schemes with a colored sidebar: some of the text would feel more incorporated into the sidebar if we used the sidebar color (pink) instead of gray. In the screenshot below I'm referring to "manage" and "free".

screen shot 2019-02-20 at 10 07 55 am

**Comment by sixhours** [comment]
@drw158 Thank you for the review! I originally had celadon as the primary color, but felt like it lacked hierarchy. That said, I was using the same green for the highlight color, so that probably contributed. 

screen shot 2019-02-21 at 4 00 39 pm

I like this iteration better, using celadon for primary, light blue for the selected color, and a darker color for the hover state. What do you think?

**Comment by torres126** [comment]
Sakura is definitely my favourite scheme :green_heart:

No visual issues spotted either!

File: assets/stylesheets/shared/_color-schemes.scss [hunk 1]
```diff
@@ -594,6 +594,135 @@
...
+
+		--color-button-primary-background-hover: #{$muriel-hot-blue-400};
+		--color-button-primary-scary-background-hover: #{$muriel-hot-red-400};
+
+		--masterbar-color: #{$muriel-white};
+		--masterbar-border-color: #{$muriel-celadon-700};
+		--masterbar-item-new-editor-background: #{$muriel-gray-500};
+		--masterbar-item-new-editor-hover-background: #{$muriel-gray-600};
+		--masterbar-toggle-drafts-editor-background: #{$muriel-gray-600};
+		--masterbar-toggle-drafts-editor-border-color: #{$gray-lighten-20};
```
Comment (blowery) [inline assets/stylesheets/shared/_color-schemes.scss]: we're trying to drop all the old color variables. Could you please use a `$muriel-gray-???` here instead?

**Comment by sixhours** [comment]
> LGTM, other than the $gray-lighten-20 usage.

Good catch! I've updated instances of those variables to use muriel values; there were a bunch of `$gray-dark` in there, too.

**Comment by blowery** [comment]
> I've updated instances of those variables to use muriel values; there were a bunch of `$gray-dark` in there, too

I think @flootr updated those in https://github.com/Automattic/wp-calypso/pull/31111 too.

**Comment by sixhours** [comment]
> I think @flootr updated those in #31111 too.

Ahh, I'll leave those out of this PR to avoid duplication. Thanks for the heads-up! (Good thing I committed changes to the wrong branch before I saw this, haha.) 🙃

**Pull request merged**